proto/ibc/applications/interchain_accounts/v1/types.proto

```diff
@@ -29,3 +29,4 @@ message InterchainAccountPacketData {
 message CosmosTx {
   repeated google.protobuf.Any messages = 1;
 }
+
```

Comment:
Side note, but I think we should rename this file to `packet.proto`. In my opinion it makes more sense and is more consistent with transfer for example. 

What do you think?

Reply:
Sounds good to me! I didn't like that vagueness of `types.proto`